lib/pitbull/strategies/static.rb

```diff
@@ -6,17 +6,13 @@ module Static
       extend ::ActiveSupport::Concern
 
       included do
-        before_action :authenticate
+        before_action :static_authorized
```

Comment:
O método ficou conjugado no passado, o correto seria no presente (indicativo).

```suggestion
        before_action :static_authorize
```

Reply:
Done!